### DISCUSSION TITLE
How to disable Grammarly extension from EditorContent

### DISCUSSION THREAD

**Discussion by ahsaanshuja** [ueberdosis/tiptap#3215] [General Questions] [answered]
Please guide me on how to disable the Grammarly extension from useEditor/EditorContent

**Comment by francistogram** [comment]
@ahsaanshuja did you end up figuring this out?

**Reply by ahsaanshuja** [reply]
Unfortunately, No

**Reply by bdbch** [reply, marked as answer]
See https://github.com/ueberdosis/tiptap/discussions/3706
